src/main/java/seedu/eduke8/exception/ExceptionMessages.java

```diff
@@ -0,0 +1,18 @@
+package seedu.eduke8.exception;
+
+public class ExceptionMessages {
+
+    public static final String ERROR_STORAGE_FAIL = "Oh no! An error has occurred when accessing the file."
+            + "Please check again!";
+    public static final String ERROR_UNKNOWN = "Oh no! An unknown error has occurred.";
+    public static final String ERROR_UNRECOGNIZED_COMMAND = "Oops! The command cannot be understood.\n"
```

Comment:
I think for all new lines we should use System.lineSeparator() instead as it will ensure that it works for any OS.

Reply:
Ok, will look out for it. Thank you!